.github/workflows/eerelease.yml

```diff
@@ -0,0 +1,124 @@
+name: Release Ansible Execution Environment
+on:
+  workflow_dispatch:
+    inputs:
+      ee-name:
+        description: >-
+          EE name if it is community-ee-base or community-ee-minimal. Example: community-ee-base
+        required: true
```

Comment:
If only certain values are accepted, `type: choice` can be used here: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs

Or, if you can always build both of these images, you can define `strategy.matrix` to easily run the same workflow for multiple images: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Reply:
I think choice is the right option here, updated it accordingly.